awsshell/wizard.py

```diff
@@ -111,20 +115,22 @@ def create_wizard(self, model):
         stages = self._load_stages(model.get('Stages'), env)
         return Wizard(start_stage, stages, env, self._error_handler)
 
+    def _load_stage(self, stage, env):
+        stage_attrs = {
+            'name': stage.get('Name'),
+            'prompt': stage.get('Prompt'),
+            'retrieval': stage.get('Retrieval'),
+            'next_stage': stage.get('NextStage'),
+            'resolution': stage.get('Resolution'),
+            'interaction': stage.get('Interaction'),
+        }
+        creator = self._cached_creator
+        interaction = self._interaction_loader
+        delegation = self._delegation_loader
+        return Stage(env, creator, interaction, delegation, **stage_attrs)
+
     def _load_stages(self, stages, env):
-        def load_stage(stage):
```

Comment:
I'm glad that you removed that. Usually it is not great practice to define a function inside a function, especially if it will be called more than once.